### PR TITLE
Instruments api

### PR DESCRIPTION
**Pull request by headStyleColorRed**
I added a list of instruments 🤓

## Routes
- [x] GET /instruments/
- [x] GET /instruments/:filterBy?

## Tests
- [x] Passing

## Screenshot
Screenshot 2022-10-12 at 17 15 06

## Suggestions
@ageddesi, could you add the hacktoberfest-accepted tag :B?

### CODE REVIEW

File: modules/instruments/api/instruments-routes.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     *       '200':
+     *         description: OK
+     *         schema:
+     *           type: array
+     *           items:
+     *             type: string
+     *             example: Saxhorn, Tuba
+     */
+    app.get('/instruments/', (req: Request, res: Response) => {
+        res.json({
```
Comment (ageddesi) [inline modules/instruments/api/instruments-routes.ts:23]: Hi, do you mind adding a qty attribute to the end of this endpoint so a user can specify the amount they would like returned. We have a route util for this as well called getQtyFromRequest

**Review by ageddesi — CHANGES_REQUESTED** [review]

**Comment by ageddesi** [comment]
HI @headStyleColorRed thank you for this, what an awesome endpoint idea. I have asked you to just make 1 small modification then its ready for merging. Thank you.

File: modules/instruments/api/instruments-routes.ts (hunk 1 above)
Comment (headStyleColorRed) [reply modules/instruments/api/instruments-routes.ts:23]: Sure :D

Comment (headStyleColorRed) [reply modules/instruments/api/instruments-routes.ts:32]: @ageddesi Should be done with the tests updated :)

**Comment by ageddesi** [comment]
Thanks @headStyleColorRed for that contribution 👍 Great work and happy Hacktoberfest

**Comment by headStyleColorRed** [comment]
> Thanks @headStyleColorRed for that contribution 👍 Great work and happy Hacktoberfest

Thank you!!!

**Pull request merged**